app/Platform/Actions/RevalidateAchievementSetBadgeEligibility.php

```diff
@@ -120,17 +119,6 @@ private function revalidateCompletionBadgeEligibility(PlayerGame $playerGame): v
             );
             PlayerBadgeAwarded::dispatch($badge);
             PlayerGameCompleted::dispatch($playerGame->user, $playerGame->game);
-
-            // TODO WriteUserActivity
```

Comment:
let's not do that for now.
adding another listener for WriteUserActivity might be relevant for when activity feeds are implemented but it won't happen here, events are already dispatched.